onfido_strings.translatable

```diff
@@ -123,8 +123,9 @@
       }
     },
     "upload_document": "$confirm.confirm$",
-    "upload_file": "$capture.upload_file_old$",
-    "take_photo": "$capture.take_photo$"
+    "upload_file": "$capture.upload_file_NEW$",
```

Comment:
Do we need the `_NEW` tag here?

Reply:
Good point. I added the `_NEW` tag to differentiate it as I was updating the copy to keep track of my changes vs @raemarielee original edits in the spreadsheet but I think it should be fine to remove it now that I've cleaned things up.